docs/standard/class-libraries-multi-targeting.md

```diff
@@ -0,0 +1,283 @@
+---
+title: .NET Class Library Multi-Targeting
+description: Learn how to create .NET class libraries for multiple .NET platforms that can access and wrap platform-specific functionality.
+keywords: .NET, .NET Core
+author: immol
+ms.author: mairaw
+ms.date: 03/19/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+ms.assetid: 375da522-3520-4735-b1de-68c7b71719d5
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+
+# Introduction
+
+Even though .NET Standard 2.0 is very rich in terms APIs, there are rare cases
```

Comment:
I would probably open up with ".NET Standard 2.0 offers an extensive API surface that can be shared across .NET applications. There are cases where you may need to access platform specific APIs that are not part of .NET (for instance..."

Reply:
Very fair. Starting with a negative is rarely a good way to sell anything :-)

---
In reply to: [175868072](https://github.com/dotnet/docs/pull/4709#discussion_r175868072) [](ancestors = 175868072)